admin/javascript/dist/silverstripe-backend.js

```diff
@@ -82,7 +82,14 @@
     }, {
       key: 'post',
       value: function post(url, data) {
-        return this.fetch(url, { method: 'post', credentials: 'same-origin', body: data }).then(checkStatus);
+        return this.fetch(url, {
+          method: 'post',
+          headers: new Headers({
+            'Content-Type': 'application/x-www-form-urlencoded'
```

Comment:
We should allow passing in the desired content type, but given that the whole "backend from scratch" approach is likely superseded by something else, I won't be too picky for now ;)